app/uk/gov/hmrc/lisaapi/config/AppContext.scala

```diff
@@ -40,4 +39,10 @@ class AppContext @Inject()(
   lazy val v2endpointsEnabled = runModeConfiguration.getBoolean("api.endpointsEnabledv2").getOrElse(throw new RuntimeException(s"Missing key api.endpointsEnabledv2"))
 
   override protected def mode = environment.mode
+
+  def endpointIsEnabled(endpoint: String): Boolean = {
+    val endpointIsDisabled = runModeConfiguration.getStringList("api.disabledEndpoints").fold(false)(list => list.contains(endpoint))
+
+    !endpointIsDisabled
```

Comment:
One line it?

Reply:
I'd considered it but the logic was a little hard to follow. Going to refactor a little to make it simpler...

Reply:
Refactored in dfc9ff6